reconstruction/rtpc/pom.xml

```diff
@@ -97,6 +97,19 @@
       <artifactId>clas-analysis</artifactId>
       <version>7.0.0-SNAPSHOT</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>swim-tools</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.ejml</groupId>
+      <artifactId>ejml-all</artifactId>
+      <version>0.40</version>
```

Comment:
This is more a comment for future reference: in the last version of the clas-tracking package (see the pull request #778), support for multiple matrix libraries, including ejml, was added. Currently there we are linking to version 0.39 and we should align the version when this branch will be merged to development.